src/pages/App.tsx

```diff
@@ -135,6 +136,7 @@ export default function App() {
               <CustomRoute exact path="/beta/migrate/:version" component={MigrateV2} layout={Layout} />
 
               <CustomRoute exact path="/beta/swap/" component={SwapV2} layout={Layout} />
+              <CustomRoute exact path="/beta/stake/" component={StakeV2} layout={Layout} />
```

Comment:
we need to support dynamic version here, like we are doing it in current staking page `/stake/:version`